src/storageClients/BlobStorageManager.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Promise = require('bluebird');
+const azure = Promise.promisifyAll(require('azure-storage'));
```

Comment:
True, there is a configuration you can add to apply promisifyAll certain functions rather than the whole library. I will add that config.